src/rust/engine/engine_pyo3/src/externs/interface/fs.rs

```diff
@@ -209,6 +211,21 @@ impl PyObjectProtocol for PySnapshot {
   fn __hash__(&self) -> u64 {
     self.0.digest.hash.prefix_hash()
   }
+
```

Comment:
This class is double declared right now (see #12422), so this also needs to be added in https://github.com/pantsbuild/pants/blob/9e781c1e86331f46986e6fb13d6e63f90a3d2159/src/rust/engine/src/externs/fs.rs#L113

Reply:
Will do.

Reply:
Done.